### ISSUE TITLE
Modules in $PSHome\modules don't load in restricted execution policy

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/PowerShell#1733]
## Steps to reproduce

ipmo psreadline
## Expected behavior

the module should load
## Actual behavior

Tells you that you need to change the execution policy.
## Environment data

``` powershell
> $PSVersionTable
Name                           Value
----                           -----
PSEdition                      Core
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
CLRVersion
PSRemotingProtocolVersion      2.3
BuildVersion                   3.0.0.0
WSManStackVersion              3.0
GitCommitId                    v6.0.0-alpha.7-378-g86c5d3e4c88fe1489dcb657132e64dd8535da6a1
SerializationVersion           1.1.0.1
PSVersion                      6.0.0-alpha

```

**Comment by mirichmo** [comment]
Check if this is a duplicate of #1716

**Comment by mirichmo** [comment]
@TravisEz13 Is this a duplicate of #1716?

**Comment by TravisEz13** [comment]
No, on Windows PowerShell these modules load even in restricted execution policy.  So it is not a duplicate.

**Comment by charub** [comment]
Closing as this is a duplicate of #1716

**Comment by TravisEz13** [comment]
These issues are not duplicate.  Built-in Powershell modules should load even with restricted policy.  Even if it is not the duplicate.

**Comment by joeyaiello** [comment]
This is fixed now on `6.0.0-beta.1`:

```powershell
Set-ExecutionPolicy Restricted
rmo psreadline
ipmo psreadline
Get-PSReadlineOption

EditMode                               : Windows
...
```
